internal/operands/tekton-pipelines/tekton-pipelines.go

```diff
@@ -128,16 +128,17 @@ func (t *tektonPipelines) Cleanup(request *common.Request) ([]common.CleanupResu
 }
 
 func isUpgradingNow(request *common.Request) bool {
-	return request.Instance.Status.ObservedVersion != environment.GetOperatorVersion()
+	return request.Instance.Status.ObservedVersion != common.GetOperatorVersion()
 }
 
 func reconcileTektonPipelinesFuncs(pipelines []pipeline.Pipeline) []common.ReconcileFunc {
 	funcs := make([]common.ReconcileFunc, 0, len(pipelines))
 	for i := range pipelines {
 		p := &pipelines[i]
 		funcs = append(funcs, func(request *common.Request) (common.ReconcileResult, error) {
-			namespace := request.Instance.Spec.Pipelines.Namespace
-			p.Namespace = namespace
+			if request.Instance.Spec.TektonPipelines != nil && request.Instance.Spec.TektonPipelines.Namespace != "" {
```

Comment:
in the same namespace as operator

Reply:
So we'll add `p.Namespace = "kubevirt"` before the `if` condition?

Reply:
@ksimon1 the **operator** or the SSP **CR**?

Reply:
@codingben , hardcoding namespace is not a good design. There is a function to get the operator namespace: `common.GetOperatorNamespace()`

Reply:
You are right, operator can be deployed in a different namespace than CR. I would vote for CR namespace, because the same way it is done for common templates